USE_CASES.md

```diff
@@ -1344,6 +1345,129 @@ func main() {
 }
 ```
 
+<a name="tutorial_deployments"></a>
+# Tutorial Deployments
+## Heroku
+
+### Require
+* [Heroku CLI](https://devcenter.heroku.com/articles/heroku-cli)
+* [godep](https://github.com/tools/godep)
+
+### Initial project with example code
+
+```go
+package main
+
+import (
+	"os"
+
+	"github.com/gin-gonic/gin"
+	"github.com/sendgrid/rest"
+	sendgrid "github.com/sendgrid/sendgrid-go"
+	"github.com/sendgrid/sendgrid-go/helpers/mail"
+)
+
+var sendgridkey = "SENDGRID_API_KEY"
+
+func main() {
+	port := os.Getenv("PORT")
+
+	if port == "" {
+		port = "5000"
+	}
+	router := gin.New()
+	router.Use(gin.Logger())
+	router.GET("/sendmail/:to", sendmail)
+
+	router.Run(":" + port)
+}
+
+func sendmail(c *gin.Context) {
+	to := c.Param("to")
+	res, err := sendmailWithSendGrid(to)
+	if err != nil {
+		c.JSON(400, err.Error())
```

Comment:
I think for the error it might make more sense to return a 500 instead of a 400. Could you use the http packages constant: `http.StatusInternalServerError`